basyx/aas/model/aas.py

```diff
@@ -52,31 +54,39 @@ class AssetInformation:
     def __init__(self,
                  asset_kind: base.AssetKind = base.AssetKind.INSTANCE,
                  global_asset_id: Optional[base.Identifier] = None,
-                 specific_asset_id: Optional[Set[base.SpecificAssetId]] = None,
+                 specific_asset_id: Optional[Iterable[base.SpecificAssetId]] = None,
```

Comment:
This could be just `Iterable[base.SpecificAssetId] = ()`

Reply:
Fixed in [f537658](https://github.com/eclipse-basyx/basyx-python-sdk/pull/149/commits/f53765889a839b96d3d9e5532b5e1d03ee1dd581)